<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KIPMsgDetailConfigDialogBase</class>
 <widget class="QDialog" name="KIPMsgDetailConfigDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>460</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>460</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>460</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>500</width>
    <height>685</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>KIpMessenger Detail &amp; Configuration</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QGroupBox" name="groupBox16">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>480</width>
     <height>410</height>
    </rect>
   </property>
   <property name="title">
    <string>Detail configuration</string>
   </property>
   <widget class="QCheckBox" name="m_ConfirmIconCheckbox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>82</y>
      <width>450</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Show minimi&amp;ze Send/Open Confirm window</string>
    </property>
    <property name="shortcut">
     <string>Alt+Z</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="m_NoHideCheckbox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>112</y>
      <width>450</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>No Hide Recieve &amp;message form when repling</string>
    </property>
    <property name="shortcut">
     <string>Alt+M</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox17">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>10</y>
      <width>123</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>Quote Char</string>
    </property>
    <layout class="QHBoxLayout">
     <item>
      <widget class="KLineEdit" name="m_QuoteCharEditbox">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QCheckBox" name="m_IMEAutomaticalyCheckbox">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>280</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>IM ON/OFF A&amp;utomaticaly</string>
    </property>
    <property name="shortcut">
     <string>Alt+U</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="m_HotKeyCheckbox">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>310</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Send/Recieve HotKey(&amp;Alt+Ctrl+S,R)</string>
    </property>
    <property name="shortcut">
     <string>Alt+A</string>
    </property>
   </widget>
   <widget class="QLabel" name="m_PreviewAbsenceIconLabel">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>332</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>default
icon</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="KPushButton" name="m_IconButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>282</y>
      <width>140</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>&amp;Icon</string>
    </property>
    <property name="shortcut">
     <string>Alt+I</string>
    </property>
   </widget>
   <widget class="KLineEdit" name="m_RecieveSoundFileNameEditbox">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>232</y>
      <width>230</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="m_OpenBySingleClickCheckbox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>172</y>
      <width>450</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>&amp;Open Send form by Single Click</string>
    </property>
    <property name="shortcut">
     <string>Alt+O</string>
    </property>
   </widget>
   <widget class="KPushButton" name="m_RecieveSoundButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>230</y>
      <width>140</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Recie&amp;ved Sound</string>
    </property>
    <property name="shortcut">
     <string>Alt+V</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="m_FireInterceptCheckbox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>142</y>
      <width>450</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Show [&amp;Fire/Intercept] as [Send/Recieve]</string>
    </property>
    <property name="shortcut">
     <string>Alt+F</string>
    </property>
   </widget>
   <widget class="KPushButton" name="m_AbsenceIconButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>350</y>
      <width>140</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>A&amp;bsence Icon</string>
    </property>
    <property name="shortcut">
     <string>Alt+B</string>
    </property>
   </widget>
   <widget class="KPushButton" name="m_PlayButton">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>230</y>
      <width>80</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Play</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="m_NonePopupOnAbsenceCheckbox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>202</y>
      <width>450</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>&amp;None popup recieve on Absence mode</string>
    </property>
    <property name="shortcut">
     <string>Alt+N</string>
    </property>
   </widget>
   <widget class="QLabel" name="m_PreviewIconLabel">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>262</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>default
icon</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="KLineEdit" name="m_AbsenceIconFileNameEditbox">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>352</y>
      <width>230</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="KLineEdit" name="m_IconFileNameEditbox">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>282</y>
      <width>230</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>420</y>
     <width>480</width>
     <height>30</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
   <widget class="KPushButton" name="m_CancelButton">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>0</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>&amp;Cancel</string>
    </property>
    <property name="shortcut">
     <string>Alt+C</string>
    </property>
   </widget>
   <widget class="KPushButton" name="m_ApplyButton">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>0</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Appl&amp;y</string>
    </property>
    <property name="shortcut">
     <string>Alt+Y</string>
    </property>
   </widget>
   <widget class="KPushButton" name="m_OKButton">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>0</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>O&amp;K</string>
    </property>
    <property name="shortcut">
     <string>Alt+K</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>KPushButton</class>
   <extends>QPushButton</extends>
   <header>kpushbutton.h</header>
  </customwidget>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>m_IMEAutomaticalyCheckbox</tabstop>
  <tabstop>m_HotKeyCheckbox</tabstop>
  <tabstop>m_ConfirmIconCheckbox</tabstop>
  <tabstop>m_NoHideCheckbox</tabstop>
  <tabstop>m_FireInterceptCheckbox</tabstop>
  <tabstop>m_OpenBySingleClickCheckbox</tabstop>
  <tabstop>m_NonePopupOnAbsenceCheckbox</tabstop>
  <tabstop>m_RecieveSoundButton</tabstop>
  <tabstop>m_RecieveSoundFileNameEditbox</tabstop>
  <tabstop>m_PlayButton</tabstop>
  <tabstop>m_IconButton</tabstop>
  <tabstop>m_IconFileNameEditbox</tabstop>
  <tabstop>m_AbsenceIconButton</tabstop>
  <tabstop>m_AbsenceIconFileNameEditbox</tabstop>
  <tabstop>m_OKButton</tabstop>
  <tabstop>m_CancelButton</tabstop>
  <tabstop>m_ApplyButton</tabstop>
 </tabstops>
 <includes>
  <include location="local">klineedit.h</include>
  <include location="local">kpushbutton.h</include>
  <include location="local">klineedit.h</include>
  <include location="local">kpushbutton.h</include>
  <include location="local">kpushbutton.h</include>
  <include location="local">kpushbutton.h</include>
  <include location="local">klineedit.h</include>
  <include location="local">klineedit.h</include>
  <include location="local">kpushbutton.h</include>
  <include location="local">kpushbutton.h</include>
  <include location="local">kpushbutton.h</include>
 </includes>
 <resources/>
 <connections>
  <connection>
   <sender>m_AbsenceIconButton</sender>
   <signal>clicked()</signal>
   <receiver>KIPMsgDetailConfigDialogBase</receiver>
   <slot>slotAbsenceIconClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_AbsenceIconFileNameEditbox</sender>
   <signal>textChanged(QString)</signal>
   <receiver>KIPMsgDetailConfigDialogBase</receiver>
   <slot>slotAbsenceIconNameChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_ApplyButton</sender>
   <signal>clicked()</signal>
   <receiver>KIPMsgDetailConfigDialogBase</receiver>
   <slot>slotApplyClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_CancelButton</sender>
   <signal>clicked()</signal>
   <receiver>KIPMsgDetailConfigDialogBase</receiver>
   <slot>slotCancelClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_OKButton</sender>
   <signal>clicked()</signal>
   <receiver>KIPMsgDetailConfigDialogBase</receiver>
   <slot>slotOkClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_PlayButton</sender>
   <signal>clicked()</signal>
   <receiver>KIPMsgDetailConfigDialogBase</receiver>
   <slot>slotPlaySoundClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_RecieveSoundButton</sender>
   <signal>clicked()</signal>
   <receiver>KIPMsgDetailConfigDialogBase</receiver>
   <slot>slotRecievedSoundClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_IconButton</sender>
   <signal>clicked()</signal>
   <receiver>KIPMsgDetailConfigDialogBase</receiver>
   <slot>slotIconClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>m_IconFileNameEditbox</sender>
   <signal>textChanged(QString)</signal>
   <receiver>KIPMsgDetailConfigDialogBase</receiver>
   <slot>slotIconNameChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
